kafka-ui-react-app/src/theme/theme.ts

```diff
@@ -50,6 +50,11 @@ const theme = {
     mainColor: Colors.neutral[5],
   },
   panelColor: Colors.neutral[0],
+  breadcrumb: Colors.neutral[30],
+  connectEditWarring: Colors.yellow[10],
```

Comment:
`connectEditWarring` looks like a typo

Reply:
It's used in `components/Connect/Edit ` and nowhere else. So I made it separatly.

Reply:
Oh sry!)) U right of course!